test/test_images/recordevents/eventstore.go

```diff
@@ -73,21 +73,36 @@ func (es *eventStore) checkAppendBlock() {
 }
 
 // Store the specified event.
-func (es *eventStore) StoreEvent(event cloudevents.Event, httpHeaders map[string][]string) {
+func (es *eventStore) StoreEvent(event *cloudevents.Event, evErr error, httpHeaders map[string][]string) {
```

Comment:
The `evErr` you receive here is not a validation error, maybe should you rename the `EventInfo.ValidationError` to just `EventInfo.Error`?

Reply:
This error is set by a receiver that is a standard http server, so it can end up getting something that results in non-validation errors (for instance ErrCannotConvertToEvent).  I prefer to record the fact that we saw such errors in cases where a test may (for instance) want to watch for handling of invalid data.

Reply:
ah, never mind, i see what you're getting at.  Yes, I'll look at renaming the field inside the EventInfo structure to something less specific than ValidationError.